_literate/1_why_Julia.jl

```diff
@@ -121,6 +122,23 @@
 # )
 # ```
 
+# Here is R {data.table}:
+
+# ```r
+# library(data.table)
+# 
+# n <- 10e3
+# dt <- data.table(
+#     x = sample(c("A", "B", "C", "D"), n, replace = TRUE),
+#     y = runif(n),
+#     z = rnorm(n)
+# )
+# 
+# bench::mark(
+#     dt[, .(median(y), mean(z)), by=.(x)]
+# )
+# ```
+
```

Comment:
We could remove this and just keep the footnote?

Reply:
Of course, in the end it is your decision and I understand if it is too verbose.

Reply:
Thanks for the understanding. data.table is awesome and *fast*. But I don't want to make this intro to Julia a "benchmark festival" 😆